.gitlab/build_tioga.yml

```diff
@@ -29,7 +29,7 @@
 # Build jobs
 tioga-clang_16_0_0_hip:
```

Comment:
Should this 16 also be changed to 18?

Reply:
@adayton1 good catch! clang18 is the default.

Reply:
Yup I'll rename the job